test/api/controllers/eventSearch.js

```diff
@@ -0,0 +1,131 @@
+let mongoose = require("mongoose");
+let Event = require('../../../api/models/Event');
+
+let chai = require('chai');
+let chaiHttp = require('chai-http');
+let server = require('../../../app');
+let should = chai.should();
+
+let token = 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiI1OWYwMDM4ODkwN2IzNzA0YjRjNDYxZmQiLCJpc3MiOiJZb3VyVGFibGUiLCJpYXQiOjE1MDg5MDE3OTl9.ot_1psqlXEdsaG64zgigJz3DWJYKS-n-QEd-9INz1Mg';
+
+chai.use(chaiHttp);
+
+describe('Events Search', () => {
+	
+	describe('/GET /events', () => {
+	it('update one event title to TORONTO', (done) => {
+		chai.request(server)
+		.put('/event/' + '59fa7b35c121b4197441860e')
```

Comment:
This will need this particular event to be created before running the tests. For unit tests can we do something like:
1. create an event
2. update this event
3. search for the event
4. maybe some other tests if needed
5. delete the event

So in this way the tests are independent and will also go through all the endpoints related to `event`

Reply:
Sounds good.
Will modify the test case.
-- I did it like this because I already tested create and update in the other file: event.js.

Reply:
Yeah, I was trying to do what you said.
However, our event/put method needs a mongo id to update an event.
 -- May be I will modify the create/event method so it will return mongo id after success?

Thanks.

Reply:
ya I think including a mongo id in the response make sense

Reply:
Cool. I will change the create/event endpoint to suit our needs.